R/appveyor.R

```diff
@@ -5,25 +5,34 @@ AppVeyorCI <- R6Class(
 
   public = list(
     get_branch = function() {
-      Sys.getenv("APPVEYOR_REPO_BRANCH")
+      self$has_env("APPVEYOR_REPO_BRANCH")
     },
     get_tag = function() {
-      Sys.getenv("APPVEYOR_REPO_TAG")
+      self$has_env("APPVEYOR_REPO_TAG")
     },
     is_tag = function() {
       self$get_tag() != ""
     },
     get_slug = function() {
-      Sys.getenv("APPVEYOR_REPO_NAME")
+      self$has_env("APPVEYOR_REPO_NAME")
     },
     get_build_number = function() {
-      paste0("AppVeyor build ", Sys.getenv("APPVEYOR_BUILD_NUMBER"))
+      paste0("AppVeyor build ", self$has_env("APPVEYOR_BUILD_NUMBER"))
     },
     get_build_url = function() {
-      paste0("https://ci.appveyor.com/project/", self$get_slug(), "/build/", Sys.getenv("APPVEYOR_BUILD_VERSION"))
+      paste0("https://ci.appveyor.com/project/", self$get_slug(), "/build/", self$has_env("APPVEYOR_BUILD_VERSION"))
     },
     get_commit = function() {
-      Sys.getenv("TRAVIS_COMMIT")
+      self$has_env("TRAVIS_COMMIT")
+    },
+    is_env = function(env, value) {
+      Sys.getenv(env, value)
```

Comment:
What are the semantics here? Maybe we need to rename to `get_env` ?

Reply:
`get_env()` only returns imo, therefore I named it `is_env()` to induce a comparison. `has_env()` just checks for its existence, i.e. not being `""`.

Reply:
The original `Sys.getenv(env) == value` looked much better to me. The way this is implemented, `is_env()` will return the env value, or `value` if unset.

Reply:
Right, it should be `Sys.getenv(env) == value`. Hm, lots of small mistakes. Good you all spot them but they are too many.